docker/Dockerfile

```diff
@@ -3,7 +3,6 @@ ARG K_VERSION
 FROM runtimeverificationinc/kframework-k:ubuntu-jammy-$K_VERSION
 
 ARG K_VERSION
-ARG PYK_VERSION
 
 # create non-root user and adjust UID:GID on start-up
```

Comment:
FWIW I would love to upstream the improvements in this dockerfile to K Framework. Not sure how many downstream repos would break if we change our current dockerfile, though.

Reply:
But even if there's breakage, a dependent Dockerfile could be fixed by changing the `ENTRYPOINT`, right?

Reply:
Most likely yes. This image is overall just more restrictive so it _should_ not be a problem to convert. The only thing I could think of is a setup that somehow requires running the container as root - which one could do but is not the default with the changed entrypoint.

Reply:
I think most of our repos have a `USER` call in their `Dockerfile`.